tests/pass/intrinsics-math.rs

```diff
@@ -1,5 +1,4 @@
-// SPDX-License-Identifier: MIT OR Apache-2.0
-// SPDX-FileCopyrightText: The Rust Project Developers (see https://thanks.rust-lang.org)
```

Comment:
These were some old license headers that we just failed to remove when most of them got removed. The license and copyright holders are the same as for the main repo so they add no information.